### PR TITLE
First attempt at a few NCRC gettings started pages

### PR DESCRIPTION
**Pull request by permcody**
## Reason

This PR is here to demonstrate what content (navigation) we should be working on for NCRC users.

## Design

This is just a few pages (mostly copied) from existing MOOSE developers pages, but more focused on the user access levels.

## Impact

Minor/None

### CODE REVIEW

**Comment by rpodgorney** [comment]
I made major changes to this PR but somehow broke it when I tried to rebase and push......I'll need some help tomorrow morning...

**Comment by rpodgorney** [comment]
git push --set-upstream origin pullrequest
remote: Permission to idaholab/moose.git denied to rpodgorney.
fatal: unable to access 'https://github.com/idaholab/moose.git/': The requested URL returned error: 403

File: modules/doc/content/ncrc/ncrc_binary.md
Comment (GiudGiud) [inline modules/doc/content/ncrc/ncrc_binary.md:229]:
```suggestion
login nodes (e.g., `ssh sawtooth2`). Change directory to the location of the script
```

File: modules/doc/content/ncrc/ncrc_ondemand.md
Comment (GiudGiud) [inline modules/doc/content/ncrc/ncrc_ondemand.md:99]:
```suggestion
If you are more comfortable using the command line than the on demand interface, you may follow the instructions [here](ncrc/ncrc_binary.md)
```

**Review by GiudGiud — CHANGES_REQUESTED** [review]
had a quick look

**Comment by rpodgorney** [comment]
Is this close?

**Comment by permcody** [comment]
Rob, Did you intend to update the large_media hash? I'm guessing not. I'll help you with this when I get off my _very long call_.

**Comment by rpodgorney** [comment]
I had added some large media files when I was working on the Falcon doco pages—but in the end the images ended up in the Falcon media folder

We can reject that change


--
Robert K. Podgorney
Chief Computational Scientist
Advanced Scientific Computing Division
Idaho National Laboratory

Office: 208-526-1524<tel:208-526-1524>
Cell: 208-520-9361<tel:208-520-9361>
email: ***@***.******@***.***>
Statement of Confidentiality
The contents of this e-mail message and any attachments are confidential and are intended solely for addressee. The information may also be legally privileged. This transmission is sent in trust, for the sole purpose of delivery to the intended recipient. If you have received this transmission in error, any use, reproduction or dissemination of this transmission is strictly prohibited. If you are not the intended recipient, please immediately notify the sender by reply e-mail or phone and delete this message and its attachments, if any.


From: Cody Permann ***@***.***>
Date: Thursday, June 24, 2021 at 1:12 PM
To: idaholab/moose ***@***.***>
Cc: Robert K. Podgorney ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] Re: [idaholab/moose] First attempt at a few NCRC gettings started pages (#18144)

Rob, Did you intend to update the large_media hash? I'm guessing not. I'll help you with this when I get off my very long call.

File: modules/doc/content/ncrc/ncrc_binary.md
Comment (permcody) [reply modules/doc/content/ncrc/ncrc_binary.md:192]: Good question: I'll do some checking but will leave this as-is for now.

**Comment by permcody** [comment]
@GiudGiud - Can we get another review and hopefully an approval? We kind of would like this site in place even if it's not perfect since the new NCRC site references it.

File: modules/doc/content/ncrc/gen_ondemand.md
Comment (GiudGiud) [inline modules/doc/content/ncrc/gen_ondemand.md:50]:
```suggestion
If you are more comfortable using the command line than the on demand interface, you may follow the instructions [here](ncrc/ncrc_binary.md)
```

**Review by GiudGiud — DISMISSED** [review]

**Comment by rpodgorney** [comment]
can someone merge this in the AM?

**Comment by GiudGiud** [comment]
I ll merge it tonight after the test suite reruns for that typo

**Review by GiudGiud — DISMISSED** [review]

**Comment by GiudGiud** [comment]
Actually @permcody can you do it tomorrow? Not sure what the final call was on this large media submodule update

**Comment by loganharbour** [comment]
This changes the repo by almost 3 MB - I'm going to move the screenshots into large_media.

**Comment by loganharbour** [comment]
Looks like you didn't even use two images; we can remove them from large_media after.

**Comment by rpodgorney** [comment]
Those images will be needed later—as the doco expands/becomes more complete


--
Robert K. Podgorney, PhD
Chief Computational Scientist
Advanced Scientific Computing Division
Idaho National Laboratory

Office: 208-526-1524<tel:208-526-1524>
Cell: 208-520-9361<tel:208-520-9361>
email: ***@***.******@***.***>
Statement of Confidentiality
The contents of this e-mail message and any attachments are confidential and are intended solely for addressee. The information may also be legally privileged. This transmission is sent in trust, for the sole purpose of delivery to the intended recipient. If you have received this transmission in error, any use, reproduction or dissemination of this transmission is strictly prohibited. If you are not the intended recipient, please immediately notify the sender by reply e-mail or phone and delete this message and its attachments, if any.


From: Logan Harbour ***@***.***>
Date: Friday, June 25, 2021 at 8:14 AM
To: idaholab/moose ***@***.***>
Cc: Robert K. Podgorney ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] Re: [idaholab/moose] First attempt at a few NCRC gettings started pages (#18144)

Looks like you didn't even use two images; we can remove them from large_media after.

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**
